RELEASES.md

```diff
@@ -81,6 +81,41 @@ Beta releases serve as a preview of the upcoming stable release and are meant fo
 
+## Experimental Modules
+When adding new features to Quickstart, particularly large or complicated features, we usually add these modules as "experimental" to indicate to site owners that these modules are not fully tested or feature complete and will likely experience significant or breaking changes. These modules should only be used by site owners that are actively tracking Quickstart development and are prepared to resolve the consequences of breaking changes. 
+
+Any modules with the experimental status bypass all of the minor release controls so that ongoing development can be released quickly through patch releases. This allows for fast development of new sites that are motivating the creation of the feature. 
+
+### Best Practices for Using Experimental Modules
+- **Ideal for**: Testing new features on staging or development sites before production deployment.
+- **Risks**: May introduce breaking changes; not recommended for critical production sites without thorough testing.
+
+### Identifying Experimental Modules
+To add a module as experimental, define the package as 'The University of Arizona - Experimental' in the module's info.yml file.
+
+### Graduating an Experimental Module to Stable
+For the overall stability of the Quickstart project, the number of Experimental modules at any time should be kept to a minimum, and effort should be made to convert Experimental Modules to Stable with each minor release.
+
+## Experimental Modules
+
+### Identifying Experimental Modules
+To add a module as experimental, define the package as 'The University of Arizona - Experimental' in the module's info.yml file.
+
+### Transition Plan for Experimental Modules
+
+To ensure the stability and reliability of Quickstart, the number of Experimental modules should be minimized, and efforts should be made to convert Experimental modules to Stable with each minor release.
+
+#### Qualifications for Updating from Experimental to Stable
+- **Feature Complete**: The module should have all planned features fully implemented.
+- **Testing on Live Websites**: Ideally, the module should be installed on live websites for at least one complete minor release cycle with all known issues resolved.
+- **Stable Parent Modules**: All parent modules of the experimental module should be in a stable state.
+
+#### Transition Plan
+- **Testing Phases**: The module should be fully tested in Probo.ci before being moved to Stable. The pre-release testing phase should incorporate complete testing on sample sites during the minor release cycle.
+- **Documentation**: Update and finalize comprehensive documentation covering the module’s features, configurations, and any known issues.
+- **Review and Approval**: Conduct a thorough review and seek approval from the development team and key stakeholders before transitioning the module to a stable release.
+
+
```

Comment:
```suggestion
## Experimental Modules
When adding new features to Quickstart, particularly large or complicated
features, we usually add these modules as "experimental" to indicate to site
owners that these modules are not fully tested or feature complete and will
likely experience significant or breaking changes. These modules should only be
used by site owners that are actively tracking Quickstart development and are
prepared to resolve the consequences of breaking changes. 

Any modules with the experimental status bypass all of the minor release
controls so that ongoing development can be released quickly through patch
releases. This allows for fast development of new sites that are motivating the
creation of the feature.

### Best Practices for Using Experimental Modules
- **Ideal for**: Testing new features on staging or development sites before
  production deployment.
- **Risks**: May introduce breaking changes; not recommended for critical
  production sites without thorough testing.

### Identifying Experimental Modules
To add a module as experimental, define the package as 'The University of
Arizona - Experimental' in the module's info.yml file.

### Graduating an Experimental Module to Stable
For the overall stability of the Quickstart project, the number of Experimental
modules at any time should be kept to a minimum, and effort should be made to
convert Experimental Modules to Stable with each minor release.

## Experimental Modules

### Identifying Experimental Modules
To add a module as experimental, define the package as 'The University of
Arizona - Experimental' in the module's info.yml file.

### Transition Plan for Experimental Modules

To ensure the stability and reliability of Quickstart, the number of
Experimental modules should be minimized, and efforts should be made to convert
Experimental modules to Stable with each minor release.

#### Qualifications for Updating from Experimental to Stable
- **Feature Complete**: The module should have all planned features fully
  implemented.
- **Testing on Live Websites**: Ideally, the module should be installed on live
  websites for at least one complete minor release cycle with all known issues
  resolved.
- **Stable Parent Modules**: All parent modules of the experimental module
  should be in a stable state.

#### Transition Plan
- **Testing Phases**: The module should be fully tested in Probo.ci before being
  moved to Stable. The pre-release testing phase should incorporate complete
  testing on sample sites during the minor release cycle.
- **Documentation**: Update and finalize comprehensive documentation covering
  the module’s features, configurations, and any known issues.
- **Review and Approval**: Conduct a thorough review and seek approval from the
  development team and key stakeholders before transitioning the module to a
  stable release.
```